source/extensions/filters/http/peer_metadata/filter.h

```diff
@@ -74,20 +74,23 @@ class MXMethod : public DiscoveryMethod {
 class PropagationMethod {
 public:
   virtual ~PropagationMethod() = default;
-  virtual void inject(Http::HeaderMap&) const PURE;
+  virtual void inject(StreamInfo::StreamInfo&, Http::HeaderMap&) const PURE;
```

Comment:
Suggest making this `const StreamInfo&` here and in `injectUpstream/Downstream`.

Reply:
done